### DISCUSSION TITLE
Using Participatory Planning for Empowerment

### DISCUSSION THREAD

**Discussion by epham36** [albertkun/21S-ASIAAM-191A#67] [Past Thinking Caps]
Mapping is often assumed to depict the world in its present state, "as it is" (depending on whose perspective you are considering). A powerful possibility for participatory mapping to be used for civil society is to turn to the community to depict the geographical space in which they want to live. Given how much power municipal governments, city planners, and public works departments have over our built environment, particpatory mapping offers a key way to increase community members' ownership and control over their built environment.

A notable example of this is the participatory mapping used to ascertain desired renovations to Zamora Park in El Monte, California, upon receiving a $3.7 million grant from the California Natural Resources Agency for renovations in order to encourage recreation and physical activity. This is especially beneficial for health and quality of life outcomes for El Monte residents, which have high levels of park need. Community workshops were held in order to obtain community input regarding what kinds of amenities to have in the park (refurbished basketball courts, new playgrounds, improved bus stops, etc.), as well as where they would like to see these amenities in the park itself. During the COVID-19 pandemic, community workshops were held over Zoom, utilizing Google Drawings in order to allow for collaborative selection of park amenities and placement of facilities. This community input was used in order to concretize the renovation plans for the park. Nonprofit organizations such as the Trust for Public Land and Active San Gabriel Valley have been instrumental in ensuring that community voices are heard when using funds to improve public spaces. 

![Screen Shot 2021-04-11 at 3 05 53 PM](https://user-images.githubusercontent.com/81584148/114323393-d6f6a080-9ad9-11eb-81ca-ca4e8e6b43dd.png) 
[Renovation Program from EPTDesign](https://eptdesign.com/zamora-park/). Note the community-identified facilities denoted on the renovation plans. 

Prior to the COVID-19 pandemic, participatory planning has also been done during in-person workshops using physical props and maps in order to truly make this process more tangible and accessible to community members.
![Planning-for-Real-136](https://user-images.githubusercontent.com/81584148/114323674-1ec9f780-9adb-11eb-86e7-856ba4595543.jpeg)

Participatory mapping and planning are key to redistributing power from government officials, city planners, and opaque bureaucracies back to the communities which are key stakeholders, and ultimately, the primary patrons of public spaces and their built environment. Especially given the racist histories of urban planning, housing zoning policies, and greenspace/freeway development, participatory methods such as these are key to not only empowering communities, but also giving them ownership over the spaces they reside in. 

A particular drawback to participatory planning and collaborative mapping is that its responsiveness to the community depends on how representative the pool of planning participants is. It is crucial to ensure that all community viewpoints are adequately represented, lest the true opinions of the community are overlooked despite the participatory nature of planning. This can be remediated by working with people who are familiar with the community to determine stakeholders and opinions that must be considered in the conversation. This can be done by reaching out to community leaders or community-based organizations that work closely with the communities in question to ensure the viewpoints of the community are captured and accounted for.

**Comment by rh5140** [comment]
Thank you for sharing visuals along with your post! I really like the participatory planning done with physical props at in-person workshops and you're right on the mark that it makes the process "more tangible and accessible to community members."

My memory is a bit fuzzy on this, but I think one of the students in a different class I'm in shared this anecdote: AI is being used in urban planning now for developers to determine what areas are "underdeveloped" to work on. My classmate asked how they would avoid contributing to gentrification that way and the developers said there was no way for them to make that distinction with the technology.

Considering the rapid growth and adoption of technologies like that, it's only more important that letting the community be an active part of decisions that will affect them becomes the norm.

**Reply by epham36** [reply]
I've never thought of AI being involved in urban planning! I've always thought that urban planning required such a human degree of intuition about space, design, and functionality that hearing about AI in that space is quite shocking. I'll have to look into that, thank you so much for bringing that up! Definitely AI can only be as "unbiased" as its algorithms and trainings -- which they hardly ever are.

**Comment by lvalles10** [comment]
WOW! Thank you so much for sharing this Emily! It's so awesome to see participatory mapping utilized in this fashion- especially since it has such positive real world effects. This reminded me of the Aquatic Center rehabilitation that's currently occurring in my hometown, San Bruno, CA. After the PG&E pipeline explosion in 2009, San Bruno received a $70 million settlement, after which the San Bruno Community Foundation (SBCF) was established to ensure the money was spent in a way that would benefit the San Bruno community. Since then, SBCF has hired dialogue facilitation groups and community input advisors to plan events that solicit community feedback. After a multi-year campaign to decide what to do with the settlement money, it was concluded that the community most wanted an Aquatic Center rehab- several maps/designs were then voted on for the final project. Here's more info from the 2015 Community Listening Campaign: https://www.sbcf.org/community-listening-campaign.

I bring up this whole story to support your claims above about the importance of ethically community-sourced data in creating public spaces that benefit towns, cities, etc. The importance of participatory mapping in these spaces cannot be understated. Participatory mapping is obviously integral to community empowerment, as you stated above, but I'd also like to add that for public projects participatory mapping is important simply for community enjoyment as well. Really appreciated your post!!

**Comment by ninalong** [comment]
Thank you for this post Emily! This is a really fascinating example of how participatory planning can empower communities to shape and take ownership over the spaces that are meant to be for their benefit. I'm interested to learn more about the virtual and in-person practices of participatory planning that were utilized for this park, because our group project is fairly similar. Our client, the KTown Neighborhood Council, is seeking to complete streetscape improvements on a series of blocks in the neighborhood, and is looking for community input on the kinds and placement of amenities. As we transition from only virtual outreach to in-person, how can we combine online and face to face methods to reach the greatest amount of people and gather the most insightful input?
